scripts/calculateRewards/proportionalPrizeContest.ts

```diff
@@ -47,3 +47,50 @@ export function calculateProportionalPrizeContest({
 
   return rewardsPerReferrer
 }
+
+export function calculateSqrtProportionalPrizeContest({
```

Comment:
i spend too long futzing with different APIs to avoid code duplication with the function above, but it wasn't obvious what the right API/erogonmics was, so I ended up with this.  we can update it once we know (or if we add more  options for the proportional prize, etc).